HULKs/hulk
# Imagine all the Images

## Introduced Changes

- `Image` -> `NaoImage`
- Remove Textual and Binary Serialization Interfaces
- Implement SerializeHierarchy for `NaoImage` to provide `.jpeg` subscription delivering JPEG compressed images

## ToDo / Known Issues

none

## Ideas for Next Iterations (Not This PR)

none

## How to Test

Subscribe to images and text data via pepsi, set parameters via pepsi. Observe that everything is fine.
